Gemfile.lock

```diff
@@ -616,69 +619,68 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.5.0)
     snaky_hash (2.0.1)
       hashie
       version_gem (~> 1.1, >= 1.1.1)
     socksify (1.7.1)
-    soft_deletion (1.6.0)
-      activerecord (>= 4.2.0, < 6.2.0)
+    soft_deletion (1.9.0)
+      activerecord (>= 5.0.0, < 7.2)
     sprockets (3.7.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
     sprockets-rails (3.4.2)
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
-    sqlite3 (1.4.2)
+    sqlite3 (1.7.2)
```

Comment:
### [sqlite3](https://github.com/sparklemotion/sqlite3-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.2 → 1.7.2

[_[change-log](https://github.com/sparklemotion/sqlite3-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/sparklemotion/sqlite3-ruby), [gem-diff](https://my.diffend.io/gems/sqlite3/1.4.2/1.7.2)_]





<details>
<summary>Commits</summary>

A change of **273** commits. See the full changes on [the compare page](https://github.com/sparklemotion/sqlite3-ruby/compare/v1.4.2...v1.7.2).

These are the first 10 commits:
- (a361f34) [Address unrelated test failures](https://github.com/sparklemotion/sqlite3-ruby/commit/a361f34e67494a39389e20e249365b43db41e627)
- (4657a51) [Bump to mini_portile2 for Ruby 3.1 compatibility](https://github.com/sparklemotion/sqlite3-ruby/commit/4657a515b8d0ccbd0541a237ea74a67e2789689e)
- (e480508) [empty commit to trigger CI](https://github.com/sparklemotion/sqlite3-ruby/commit/e48050867a153f7b41f4f4c6b36d348bdeecc460)
- (07c0950) [Downcase type values loaded from extension.](https://github.com/sparklemotion/sqlite3-ruby/commit/07c0950cedafebd5032fb00d2dae11780d2b1f4b)
- (09f0276) [Merge pull request #315 from petergoldstein/feature/add_ruby…](https://github.com/sparklemotion/sqlite3-ruby/commit/09f027628d9e271c3f58fdd48aaa5743677fa462)
- (40fb6cc) [Add a `strict: true` option.](https://github.com/sparklemotion/sqlite3-ruby/commit/40fb6cc07d0084a2dd7dec1dc2476027a3de2dc1)
- (a5d9a90) [Merge pull request #317 from casperisfine/db-config](https://github.com/sparklemotion/sqlite3-ruby/commit/a5d9a90a497d253b8105e8d7d4017aa5fb2e90d9)
- (cfecd89) [version bump to v1.4.3](https://github.com/sparklemotion/sqlite3-ruby/commit/cfecd89ac09a8df5ebbd02f330fe0458e8341c85)
- (d92bb65) [Update GitHub Actions workflows](https://github.com/sparklemotion/sqlite3-ruby/commit/d92bb65724e950e565f9827ddec84aec6ed96b3e)
- (ccfa736) [ci: update concurrency/"on" to match some best practices](https://github.com/sparklemotion/sqlite3-ruby/commit/ccfa73628e3d29010b26e1acd2acd0c2cb1736bc)


</details>